docs/branch-harvest/2026-05-03-release-v1.1.0-harvest-inventory.md

```
@@ -0,0 +1,188 @@
+
+# Archscope release/v1.1.0 harvest inventory
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Branch Evidence](#branch-evidence)
+- [Candidate Table](#candidate-table)
+- [Unsafe Import Notes](#unsafe-import-notes)
+- [Selected Implementation Evidence](#selected-implementation-evidence)
+- [Deferred Candidate Evidence](#deferred-candidate-evidence)
+- [Closeout Recommendation](#closeout-recommendation)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)